seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sink/writer/AbstractWriteStrategy.java

```diff
@@ -237,7 +236,7 @@ public void beginTransaction(Long checkpointId) {
      */
     public List<String> getTransactionIdFromStates(List<FileSinkState> fileStates) {
         String[] pathSegments = new String[]{textFileSinkConfig.getPath(), Constant.SEATUNNEL, jobId};
-        String jobDir = String.join(File.separator, pathSegments) + "/";
+        String jobDir = String.join("/", pathSegments) + "/";
```

Comment:
Because the `File.separator` that in Windows is `\`, so the path be joined it will be like this:

`/tmp/seatunnel/dist\xxxx.txt`

this path hdfs protocol is not recognized, I have tried to change the path to the windows format like `\tmp\seatunnel\dist\xxx.txt`, but the hdfs protocol still can not recognize. So in order to run the test cases related to the file connector in the windows environment I decided to change it. 

The purpose of this pr is to be able to debug better in the windows environment, there is no impact on the normal function.

If you think it is not necessary, we can close it.

Reply:
You can replace the `/` of the `path` sink option with `File.separator`

Reply:
Yup, you are right. Good idea，yyds 👍

Reply:
> `File.separator` is supported for window, whether it is the problem of the following "/"?

I will do a check. Thanks for your good advice.

Reply:
![image](https://user-images.githubusercontent.com/36807946/193575255-cdcfd1e1-f8da-4a07-8871-8277d271cfd0.png)
Please change FileSystemUtils#getFileSystem

> FileSystem fileSystem = FileSystem.get(new File(path).toPath().toUri(), CONF);

you can see https://stackoverflow.com/questions/51352404/how-can-i-get-a-proper-uri-in-java-windows